etc/ci.sh

```diff
@@ -131,7 +131,7 @@ GAPInput
     ;;
 
   testmanuals)
-    $GAP $SRCDIR/tst/extractmanuals.g
+    $GAP -O -A $SRCDIR/tst/extractmanuals.g
```

Comment:
Could you maybe add a comment above this line, saying that the purpose of the `-O` is to test that GAP loads with obsoletes disabled, and `-A` is used is because some auto-loaded packages still use obsoletes.

That way it is clear why these options are being used here, and it's clear that it's not arbitrary/accidental. It is useful for someone wondering when/whether `-A` can be removed.

Reply:
Done, good suggestion @wilfwilson !